automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/ClusterFixtureSpec.scala

```diff
@@ -99,15 +103,15 @@ abstract class ClusterFixtureSpec extends fixture.FreeSpec with BeforeAndAfterAl
     logger.info("beforeAll")
     if (!debug) {
       sys.props.get(gpallocProjectKey) match {
-        case Some(billingProject) => createRonCluster(GoogleProject(billingProject))
+        case Some(billingProject) =>
+          createRonCluster(GoogleProject(billingProject))
         case None                 => throw new RuntimeException("leonardo.billingProject system property is not set")
       }
     }
 
   }
 
   override def afterAll(): Unit = {
-    logger.info("afterAll")
```

Comment:
This statement has been useful to me in the past to figure out the order of `before/afterAll` calls :)